### PR TITLE
feat (dashpay): support dpp 0.23 with new fee structure

### PR DESCRIPTION
**Pull request by HashEngineering**
## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

**Comment by HashEngineering** [comment]
CI is broken because this relies on a missing key that was replaced by accident.

File: wallet/src/de/schildbach/wallet/Constants.java [hunk 1]
```diff
@@ -318,7 +318,7 @@ public void log(final String message) {
     public static final boolean SUPPORT_BOTH_BACKUP_WARNINGS = false;
 
     // 100,000,000 credits
-    public static final Coin DASH_PAY_FEE = Coin.parseCoin("0.01");
+    public static final Coin DASH_PAY_FEE = Coin.parseCoin("0.15");
```
Comment (Syn-McJ) [inline wallet/src/de/schildbach/wallet/Constants.java:321]: Perhaps the comment needs changing too

**Review by Syn-McJ — APPROVED** [review]
Looks good

File: wallet/src/de/schildbach/wallet/Constants.java (hunk 1 above)
Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/Constants.java:321]: good call.

**Pull request merged**